### PR TITLE
Add logic for checking time in shift form

### LINKED ISSUE #741: Checks need to be added in shift forms.
## Description
As a user,
I need to be notified if I fill invalid values in forms,
so that I can ensure correct assignment of shift.

## Mocks
we write some logic to check this https://github.com/systers/vms/blob/develop/vms/shift/forms.py#L24 and also that shift date lies in between of the start and end date of job.

### Update [Required]
 - [ ] Logic for checking start and end time of shift

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentor.

## Estimation
30 mins?

### PR DESCRIPTION
**Pull request by Monal5031**
# Description
Added logic for checking start and end time in shift app.

Fixes #741 

# Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Monal5031** [comment]
Will fix the build asap.

**Comment by Monal5031** [comment]
Done @abhishekarya286 @naman1901 @pavankm Please review.

File: vms/shift/forms.py [hunk 1]
```diff
@@ -20,6 +20,17 @@ class Meta:
             'state', 'city', 'address', 'venue'
         ]
 
+    # Shifts are bound to end on the same date so
```
Comment (pavankm) [inline vms/shift/forms.py]: would this work if my shift spans two dates?
start before midnight and end after?

Comment (Monal5031) [reply vms/shift/forms.py]: For that we need to create 2 separate shifts.

Comment (naman1901) [reply vms/shift/forms.py]: I think what @pavankm means that say my shift starts at 10PM on day 1 and goes on till 6 AM the next day. Will it work in that situation?

Comment (naman1901) [reply vms/shift/forms.py]: If `start_time` and `end_time` are datetime objects or UNIX epochs, this approach will work as it is. If not, it needs to be verified.

Comment (Monal5031) [reply vms/shift/forms.py]: Yeah I confirmed this in vms meeting, the shift has to end on the same day.
So the case you mentioned will have 2 shifts, 1 from 10pm to 11:59pm and 2nd from 00:00 to 06:00.
Just to confirm again I'll tag vms people here.
@ayushgarg1804 @abhishekarya286 @kamsuri @anjali-dhanuka

Comment (naman1901) [reply vms/shift/forms.py]: Alright, looks good then!

**Comment by Monal5031** [comment]
ping @naman1901 @MehaKaushik @ayushgarg1804 Please review.

**Pull request merged**